src/program/snabbvmx/tests/nexthop/selftest.sh

```diff
@@ -54,7 +57,8 @@ fi
 TIMEOUT=20
 COUNT=0
 while true; do
-    output=$(./snabb snabbvmx query | grep "next_hop_mac")
+    $(./snabb snabbvmx query > $TMPFILE)
```

Comment:
I don't really understand how this could make anything better.  If we keep the change then the `$()` on this line are not needed.

Reply:
I will remove this patch as it was properly fixed in #1031